draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss

```diff
@@ -1,7 +1,7 @@
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/design-tokens/sass/typography";
-@import "~@kaizen/deprecated-component-library-helpers/styles/type";
```

Comment:
Oh, is this not an actual package install?

Reply:
I believe its being pulled in from component-library, which has a dependency on that